tutorial/sample-problems/VectorCalc/VectorLineSegment2.pg

```diff
@@ -41,14 +42,14 @@ $ans = Compute("$P + $speed * t * $disp / norm($disp)");
 
 #:% section = statement
 BEGIN_PGML
-A particle starts at the point [`P = [$P]`] when [`t = 0`] and moves along a
-straight line toward [`Q = [@ Point($P + $disp) @]`] at a speed of [`[$speed]`]
-centimeters per second.  Assume that [`x`], [`y`], and [`z`] are measured in
-centimeters.
+A particle starts at the point [`P = [$P]`] when [`t = 0`]
+and moves along a straight line toward [`Q = [@ Point($P + $disp) @]`]
+at a speed of [`[$speed]`] centimeters per second . Assume that [`x`],
+[`y`], and [`z`] are measured in centimeters.
 
 Find a vector equation for the position of the object.
 
-[` \vec{r}(t) = `] [____]{$ans}
+[` \vec{r}(t) = `] [__]{$ans}
```

Comment:
Why did you change this?  You made the answer rule shorter.  If anything the answer rule should be made longer for this problem.  Also it would be better to use the `[_]{$ans}{10}` syntax for the answer rule length.  It is clearer what the length setting is than having to count underscores.  Note that this only affects the answer rule length when MathQuill is not in use.